tests/ui-toml/nonstandard_macro_braces/clippy.toml

```diff
@@ -3,4 +3,5 @@ standard-macro-braces = [
     { name = "quote::quote", brace = "{" },
     { name = "eprint", brace = "[" },
     { name = "type_pos", brace = "[" },
+    { name = "printlnfoo", brace = "[" },
```

Comment:
The bug in Ruma was that the name in the config was prefix-matched, no? Shouldn't there be a test that using `println` in the config doesn't trigger for `printlnfoo`, rather than the other way around?

Reply:
Yes, good catch!